src/employees.js

```diff
@@ -29,27 +31,67 @@ const referenceValues = {
 }
 
 export const EmployeeList = () => (
-    <List>
-        <Datagrid rowClick="edit">
-            <TextField source="internalId" label="Internal Id"/>
-            <TextField source="firstName" />
-            <TextField source="lastName" />  
-            <EmailField source="personalEmail" />
-            <TextField source="phone" />
-            <TextField source="address" />
-            <TextField source="city" />
-            <TextField source="state" />
-            <TextField source="zip" />
-            <TextField source="country" />
-            <DateField source="birthDate" />
-            <DateField source="createdAt" />
-            <DateField source="modifiedAt" />
-            <EditButton variant="outlined" />
-            <ShowButton variant="outlined" />
-        </Datagrid>
+    <List sort={{ field: 'internalId', order: 'ASC' }}
+        perPage={20}
+        pagination={false}
+        component="div"
+        actions={false}>
+        <TopToolbar sx={{ minHeight: { sm: 56 } }}>
+            <CreateButton />
+            <ExportButton />
+        </TopToolbar>
+        <EmployeeCards />
     </List>
 );
 
+const EmployeeCards = () => {
+    const { data, isLoading } = useListContext();
+    if (isLoading) {
+        return null;
+    }
+    return (
+        <Grid container spacing={1} sx={{ marginTop: '1em' }}>
+            {data.map(record => (
+                <RecordContextProvider key={record.internalId} value={record}>
+                    <Grid
+                        xs={2}
+                        item
+                    >
+                        <Card sx={{ maxWidth: 300 }}>
+                            {/*Profile image hardcoded until photo upload feature is in palce*/}
+                            <CardMedia
+                                component="img"
+                                height="200"
+                                image="https://bolavip.com/__export/1625160988556/sites/bolavip/img/2021/07/01/gettyimages-1323568516_crop1625160987586.jpg_1159711837.jpg"
```

Comment:
I updated the image to our s3, please update de url: https://entropay-assets.s3.amazonaws.com/messi.jpg

Reply:
Done.